app/controllers/employees_controller.rb

```diff
@@ -4,6 +4,14 @@ class EmployeesController < ApplicationController
   def index
   end
 
+  def submitted_forms
+    @submissions = current_user.submissions
+  end
+
+  def show
+    @submissions = current_user.submissions.where(form_id: params[:form_id])
+  end
```

Comment:
wrong action show must be used for showing user information not form information